### PR TITLE
Fix #19 install instructions

### LINKED ISSUE #19: Install Instructions Fixes
The install instructions need the following corrections:
- The `wstool merge ...` command points to a non existent file
- The step under "Be sure to install gazebo_ros_packages." is a repeat from the  step above for installing just gazebo.  The rosdep step below should now make this unnecessary
- Somewhere in there it should state that this software uses gazebo 7.0 
- Use catkin tools instead of catkin_make

### PR DESCRIPTION
**Pull request by jrgnicho**
This PR addresses the following
- Updated the install and running instructions in the README.
- Minor tweaks to the environment variable setup script.
- Can build workspace with catkin tools now.
@marip8  please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,69 +5,96 @@ Robotic product handling simulation
...
+	source devel/setup.bash
+	```
 
 ### View the URDF
-```
-roslaunch urdf_tutorial display.launch model:=`rospack find gilbreth_support`/urdf/gilbreth.xacro
-```
+
+	```
+	roslaunch urdf_tutorial xacrodisplay.launch model:=$(rospack find gilbreth_support)/urdf/gilbreth.xacro
```
Comment (marip8) [inline README.md]: The launch file has actually changed to just `display.launch`

File: README.md [hunk 2]
```diff
@@ -5,69 +5,96 @@ Robotic product handling simulation
...
+	cd ~/catkin_ws
+	source devel/setup.bash
+	```
 
 ### View the URDF
-```
-roslaunch urdf_tutorial display.launch model:=`rospack find gilbreth_support`/urdf/gilbreth.xacro
-```
+
+	```
```
Comment (marip8) [inline README.md]: For some reason your code block didn't register in the markdown. I think you might have to remove the indent to get it to work

File: README.md [hunk 3]
```diff
@@ -5,69 +5,96 @@ Robotic product handling simulation
...
+	rosservice call /gilbreth/conveyor/control "state: power: 100.0"
+	```
+    The "power" can range from 0.0 to 100.0.  Use 0.0 to stop the conveyor
+
+1. Start the part spawner:
+	
+	```
+	rosservice call /start_spawn "{}"
+	```
+    This command will make parts appear on the conveyor at random intervals.
```
Comment (marip8) [inline README.md]: Maybe worth mentioning its configurable via .yaml file?

**Comment by marip8** [comment]
Just a few minor comments, but everything installed, built, and ran correctly from scratch for me.

Somewhat related comment: maybe it would be cleaner and would add more functionality/usability if the perception code was made into a library

File: README.md (hunk 3 above)
Comment (jrgnicho) [reply README.md]: What's configurable?

Comment (marip8) [reply README.md]: The comment that parts will appear on the conveyor at random seems like the user doesn't have any control over those parts. Maybe it's worth mentioning that those parts and their associated randomized parameters are configurable

**Comment by jrgnicho** [comment]
@marip8 your comments have been addressed.

**Pull request merged**